tests/ui-toml/undocumented_unsafe_blocks/undocumented_unsafe_blocks.rs

```diff
@@ -703,4 +703,24 @@ mod issue_11709_regression {
     //~[disabled]^ undocumented_unsafe_blocks
 }
 
+fn issue_13039() {
+    unsafe fn foo() -> usize {
+        1234
+    }
+
+    fn bar() -> usize {
+        1234
+    }
+
+    // SAFETY: unnecessary_safety_comment incorrectly triggers here
+    _ = unsafe { foo() };
+
+    // SAFETY: unnecessary_safety_comment incorrectly triggers here
+    _ = bar();
+    //~^ unnecessary_safety_comment
+
+    // SAFETY: unnecessary_safety_comment incorrectly triggers here
+    _ = unsafe { foo() }
```

Comment:
nit: should be past tense since it's fixed now, and the second one is not incorrect